config/assisted-service/assisted-installer-sso.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Unless we intend on moving away from all of the stuff in `deploy/`...I am hesitant to recommend including all of the kustomize stuff in the source tree.

It isn't immediately clear if we need all of the bundle manifests included for midstream/downstream, but that isn't a problem to be solved now :sunglasses: 

**tl;dr** either we include all of the manifests in the source tree (kustomize based deployment) and move away from the python deployment mechanisms or we generate the kustomize based manifests only when we need them to build a bundle.

Reply:
I've removed the files.